src/Controller/TrickController.php

```diff
@@ -23,12 +27,29 @@ public function getAllTricks(TrickRepository $trickRepository)
     }
 
     #[Route('/trick/{id}', name: 'trick')]
-    public function getTrick(TrickRepository $trickRepository, int $id)
+    public function getTrick(Request $request, EntityManagerInterface $entityManager, TrickRepository $trickRepository, CommentRepository $commentRepository, int $id)
     {
         $trick = $trickRepository->findOneBy(['id' => $id]);
+        $comments = $commentRepository->findBy(['trick' => $id]);
+        dump($comments);
+
+        $comment = new Comment();
+
+        $form = $this->createForm(CommentType::class, $comment);
+
+        $form->handleRequest($request);
+        $comment->setTrick($trick);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $entityManager->persist($comment);
```

Comment:
you can remove this line by adding cascade persist on the $comments attribute in the Trick entity. Lets check this next session